<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главноя окно</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainVerticalLayout">
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupStorage">
        <property name="title">
         <string>Склад</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="btnShowStorage">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Склад</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNewImport">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Добавить импорт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNewExport">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Добавить экспорт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnShowImport">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Покозать импорт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnShowExport">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Покозать экспорт</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupProduct">
        <property name="title">
         <string>Товары</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="btnShowProduct">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Покзозать товары</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddProduct">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Добваить новый товар</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupOrder">
        <property name="title">
         <string>Заказы</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="btnAddOrder">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Добавить заказ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnShowOrder">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Покозать закзы</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAbout">
        <property name="text">
         <string>О программе</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>Выйти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="inlineWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="inlineVerticalLayout"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
